pkg/client.go

```diff
@@ -1212,21 +1212,28 @@ func (o *Client) ValidateAPIKey(apiKeyToken string) (*models.APIKeyValidation, e
 // public methods for misc functionality
 
 // CreateAccessToken creates an access token.
-func (o *Client) CreateAccessToken(userID uuid.UUID, durationInMinutes int) (*models.AccessToken, error) {
+func (o *Client) CreateAccessToken(userID uuid.UUID, durationInMinutes int, createAccessTokenOptions ...models.CreateAccessTokenOptions) (*models.AccessToken, error) {
 	urlPostfix := "access_token"
 
 	// assemble body params
 
 	type CreateAccessToken struct {
-		UserID            uuid.UUID `json:"user_id"`
-		DurationInMinutes int       `json:"duration_in_minutes"`
+		UserID               uuid.UUID  `json:"user_id"`
+		DurationInMinutes    int        `json:"duration_in_minutes"`
+		ActiveOrgId          *uuid.UUID `json:"active_org_id,omitempty"`
+		WithActiveOrgSupport *bool      `json:"with_active_org_support,omitempty"`
```

Comment:
For now, want to leave WithActiveOrgSupport out and just derive it from whether ActiveOrgId is nil or specified?